### PR TITLE
Slapstick stuff: blood burstin', light-lookin'

### PR DESCRIPTION
**Pull request by PKPenguin321**
## About The Pull Request
Some stuff that was spitballed in Discord by a fella that goes by Aethernaut.
- Having too much blood can make you explode into gibs (your bodyparts and belongings are preserved). This idea is stolen from goon, and shouldn't be overpowered because it takes significantly more time than just grabbing a saw and beheading a fool
- Examining a light that is turned on while you're adjacent flashes you (just visual, no stun).
- Adds a suicide to the surgical drill, you jam it into yourself and it makes you spin real fast for a few seconds before being ripped apart (gibbing, drops brain/items). Here's a video: https://file.house/Yebd.mp4

Other ideas NOT present in this PR:
- Get stuck in conveyor belts (by your shoelaces? not sure how this would even work) and get toothpasted

## Why It's Good For The Game
- Slapstick is funny
- Lethality and danger good
- Realism (good for RP!)

## Changelog
:cl:
add: Examining lights that are still on while you're too close is more realistic.
add: Having too much blood can make you explode. This is more... realistic, too.
add: Suiciding with surgical drills is also very realistic.
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
Just blood or other reagents?

Also how does this interact with the vampire race

**Comment by PKPenguin321** [comment]
just blood and i checked vamp code, they set you to MAXIMUM_BLOOD which is the highest safe blood value, shoooould be fine

also, added drill suicide

**Comment by MrDoomBringer** [comment]
suiciding ontop of a conveyor belt should do the toothpaste thing if you have shoes on

**Comment by CRITAWAKETS** [comment]
This is absolutely horrible, what even made you want to code this?

Because i want whatever did it.

Please never stop, and merge this immediately.

**Comment by HomocidlePancakes** [comment]
Cursed heart nerf

**Comment by PKPenguin321** [comment]
Cursed heart caps at max safe blood the same way vampires do, should be unchanged
I should really get a video of the drill suicide it looks real funny

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -12,6 +12,8 @@
 #define MOVE_INTENT_RUN  "run"
 
 //Blood levels
+#define BLOOD_VOLUME_MAX_LETHAL		2150
```
Comment (PKPenguin321) [inline code/__DEFINES/mobs.dm]: Blood.dm has to cap blood at this value instead of at the old max

**Comment by PepperPrepper** [comment]
I personally think it would be much more slapstick if the drill was shoved into the forehead instead of the chest. But that's just me

**Comment by PKPenguin321** [comment]
sure

**Comment by PKPenguin321** [comment]
### Here's a video of the drill suicide: https://file.house/Yebd.mp4

**Comment by cacogen** [comment]
sgood

**Comment by ninjanomnom** [comment]
Split this up into multiple prs

**Comment by PKPenguin321** [comment]
sure thing boss

**Pull request closed without merging**